### PR TITLE
Release Prep for 4.5.9

### PR DESCRIPTION
**Pull request by tony-landreth**
Issue: [sc-16495]

### CODE REVIEW

File: docs/content/releases/4.5.9.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: "4.5.9"
+date:
+draft: false
+weight: 62
+---
+
+TODO
```
Comment (tony-landreth) [inline docs/content/releases/4.5.9.md:8]: Need to supply copy.

File: docs/content/releases/4.5.8.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ The PostgreSQL Operator is released in conjunction with the [Crunchy Container S
 
 PostgreSQL Operator 4.5.8 release includes the following software versions upgrades:
 
-- [PostgreSQL](https://www.postgresql.org) versions 13.8, 12.12, 11.17, and 10.22 are now available.
+- [PostgreSQL](https://www.postgresql.org) versions 13.9, 12.13, 11.18, and 10.23 are now available.
```
Comment (tjmoore4) [inline docs/content/releases/4.5.8.md:14]: :wrench: 
I don't think we want to update these release notes, so I think this change should be reverted.

File: pkg/apis/crunchydata.com/v1/doc.go [hunk 3]
```diff
@@ -3,7 +3,6 @@ Crunchy PostgreSQL Operator API
 
 The Crunchy PostgreSQL Operator API defines HTTP(S) interactions with the Crunchy PostgreSQL Operator.
 
-
```
Comment (tjmoore4) [inline pkg/apis/crunchydata.com/v1/doc.go:6]: :question: Same as on the 4.6.x update, updates look good, question about the whitespace changes.

**Review by tjmoore4 — COMMENTED** [review]
Looks good, one question and waiting for release note update.

File: docs/content/releases/4.5.8.md (hunk 2 above)
Comment (tony-landreth) [reply docs/content/releases/4.5.8.md:14]: Sure don't.

**Review by tjmoore4 — APPROVED** [review]

**Pull request merged**